### ISSUE TITLE
about dataset

### ISSUE THREAD

**Issue by njnj0314** [nicolasdonati/DUO-FM#3]
Hello, may I ask if there is a training dataset with the top 79 of faust_a and the top 51 of scape_a? thanks

**Comment by nicolasdonati** [comment]
Hi,
There is no such dataset at the moment, it is only a test set since the goal was to test with different triangulation
If you want to try training on those shapes you can pick the test shapes as a train set :) , but be aware that sometimes exhibit  non-manifoldness, *which prevents the computation of complex functional maps*. So you would only be able to train without the complex block (n_cfmap = 0 in the config).
